api/resources_portal/emailer.py

```diff
@@ -7,6 +7,38 @@
 
 import boto3
 
+EMAIL_SOURCE = (
+    f"Resources Portal Mail Robot <no-reply@{settings.AWS_SES_DOMAIN}>"
+    if settings.AWS_SES_DOMAIN
+    else "Resources Portal Mail Robot"
+)
+EMAIL_SOURCE_TEMPLATE = "Resources Portal Mail Robot <no-reply@{}>"
```

Comment:
It might make sense to use this template in line 11 above, just so we have one unified source for the "Resources Portal Mail Robot" text.